app/__init__.py

```diff
@@ -22,13 +22,17 @@ def create_app(test_config=None):
         app.config["SQLALCHEMY_DATABASE_URI"] = os.environ.get(
             "SQLALCHEMY_TEST_DATABASE_URI")
 
-    # Import models here for Alembic setup
+    
     from app.models.task import Task
     from app.models.goal import Goal
 
     db.init_app(app)
     migrate.init_app(app, db)
 
-    # Register Blueprints here
+    
+    from .routes import tasks_bp
+    app.register_blueprint(tasks_bp)
+    from .routes import goals_bp
+    app.register_blueprint(goals_bp)
```

Comment:
you can save a line or two of code by combining the imports 
```suggestion
    from .routes import tasks_bp, goals_bp
    app.register_blueprint(tasks_bp)
    app.register_blueprint(goals_bp)
```